mozdownload/scraper.py

```diff
@@ -332,10 +335,16 @@ def show_matching_builds(self, builds):
 class DailyScraper(Scraper):
     """Class to download a daily build from the Mozilla server."""
 
-    def __init__(self, branch='mozilla-central', build_id=None, date=None,
+    def __init__(self, build_id=None, date=None,
```

Comment:
So you have removed this option to the daily scraper? It has to be kept, because other branches should still be able to be passed in. Also we have to update the scraper for tinderbox builds.